docs/rules/one-var.md

```diff
@@ -46,6 +63,15 @@ The following patterns are considered warnings:
 function foo() {
```

Comment:
> Maybe I'll just lump const into the first example and screw the foobar terminology?

Sorry, just meant the second occurrence of the sentence. Examples look good :ok_hand: